app/views/search/all_content.html.erb

```diff
@@ -3,12 +3,12 @@
 <div class="container">
   <% if  @nodes.all?{|k,v| v.blank?} %>
     <h4>
-      No Content found for <%= params[:query] %> :-(
+      No Content found for <%= CGI.unescape(params[:query]) %> :-(
```

Comment:
@gauravano this is what has been done to fix the string. In every tab. `params[:query]` is displayed. To make it universal for all tabs, we can make a function in the controller and return `CGI.unescape(params[:query])` from it, kind of like what has been done in the review, What do you think?

Reply:
this line https://github.com/publiclab/plots2/blob/master/app/views/search/_nav.html.erb#L3 is the one which is calling the helper method generating all the nav links so just changing here from CGI.unescape to url_encode will do all the work. Try it out. T

Reply:
Does this work now?

Reply:
Yes!

Reply:
Is it good to be merged?